_docs/concepts/network-and-auth/auth.md

```diff
@@ -130,17 +146,17 @@ In this setup, Istio CA is able to provide keys and certificates management for
 
 ## Future work
 
-*   Fine-grained authorization and auditing
+*   Inter-cluster service-to-service authentication
 
-*   Secure Istio components (Mixer, Pilot, etc.)
+*   Powerful authorization mechanisms: ABAC, RBAC, etc
 
-*   Inter-cluster service-to-service authentication
+*   Per-service auth enablement support
 
-*   End-user to service authentication using JWT/OAuth2/OpenID_Connect
```

Comment:
why is that gone (JWT mention) ?

Reply:
It's not only jwt, but also others, just want to make this section more concise.

Reply:
it's mentioned in the gdoc link above yet I think from a searchability/discoverability point of view it'd be nice to have JWT listed explicitly somewhere on this page

Reply:
sg, reverted.